### PR TITLE
Added config for the dev shared-kafka msk cluster

### PR DESCRIPTION
**Pull request by catalin-ilea**

### CODE REVIEW

**Review by rcrowe — APPROVED** [review]

File: dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+resource "kafka_topic" "pubsub_examples" {
```
Comment (rcrowe) [inline dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf:1]: Do you need this? We don't have it for `kafka-shared`, is there anything different about this example or was it for testing purposes?

File: dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  name               = "pubsub.pubsub-examples"
+  replication_factor = 3
+  partitions         = 10
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf:4]: I think 1 partition should be enough.

File: dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf [hunk 3]
```diff
@@ -0,0 +1,33 @@
+resource "kafka_topic" "pubsub_examples" {
+  name               = "pubsub.pubsub-examples"
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf:2]: wondering if we should just name it `pubsub.examples`. WDYT?

File: dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf (hunk 2 above)
Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/pubsub/pubsub-examples.tf:4]: Cool, let's keep it with 10 then.

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**